pubtools/pulplib/_impl/model/unit/modulemd.py

```diff
@@ -62,3 +64,35 @@ class ModulemdUnit(Unit):
 
     .. versionadded:: 2.6.0
     """
+    artifacts = pulp_attrib(
+        default=None,
+        type=list,
+        converter=frozenlist_or_none_converter,
+        pulp_field="artifacts",
+    )
+    """List of artifacts related to the given module, usually it's a list
+    rpms in nevra format without '.rpm' extension and consist of binary, debug and source
+    rpms as well.
+
+    Example:
+    ["perl-version-7:0.99.24-441.module+el8.3.0+6718+7f269185.src",
```

Comment:
Have you checked whether it renders correctly in the docs?

Doesn't really matter though, as I will do some docs revision after this is submitted.

Reply:
I haven't checked that unfortunately :(  
Let's fix it after submission, if it doesn't work.